oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/ClusterNodeInfo.java

```diff
@@ -240,6 +240,9 @@ static RecoverLockState fromString(String state) {
     /** OAK-10281 : default millis to delay a recovery after a lease timeout */
     static final long DEFAULT_RECOVERY_DELAY_MILLIS = 0;
 
+    /** OAK-11246 : default millis for perflogger info */
+    static final long DEFAULT_PERFLOGGER_INFO_MILLIS = Long.MAX_VALUE;
+
```

Comment:
don't think this belongs here, as it has nothing to do with ClusterNodeInfo. I would suggest to move it to a more generic place, eg to DocumentNodeStoreService

Reply:
Done. Moved var to DocumentNodeStoreService.